### DISCUSSION TITLE
Access register information in Master DB

### DISCUSSION THREAD

**Discussion by avazon** [oqtane/oqtane.framework#2737] [Q&A] [answered]
I have multiple tenants with different databases. But a set of information is common and I would like to have it stored in the "master db". How can I access the master repository from scheduled jobs run for the tenants databases?

**Comment by avazon** [comment, marked as answer]
I think I found the solution. Through ISiteRepository I can access all sites and repos.

**Comment by leigh-pointer** [comment]
@avazon yes that is correct
________________________________
